Marlin/Configuration.h

```diff
@@ -352,9 +352,10 @@
  *
  * Temperature sensors available:
  *
- *    -4 : thermocouple with AD8495
+ *    -5 : thermocouple with MAX31865 (only for sensors 0-1)
```

Comment:
Sorry for being late.
MAX31865 is not for "thermocouplers". It's for "platinum resistance temperature detectors" (PT100 / PT1000)

Reply:
I have to define both #define MAX6675_SS_PIN 49 and #define CS_PIN 49 or else I get errors, then even so, it works for sensor 0 but with lots of warnings on serial:
`MAX6675 Error:Temp measurement error! 
MAX6675 Error:Temp measurement error!`